internal/controllers/reports.go

```diff
@@ -13,7 +14,35 @@ type ReportsServer struct {
 	store datastore.ReportStore
 }
 
-func (r *ReportsServer) GetMemberCounts(w http.ResponseWriter, req *http.Request) {
+func (r *ReportsServer) GetAccessStatsChart(w http.ResponseWriter, req *http.Request) {
+	resourceName := req.URL.Query().Get("resourceName")
+	day := req.URL.Query().Get("day")
+
+	var d time.Time
+	var err error
+
+	if len(resourceName) == 0 {
+		http.Error(w, "please provide a resourceName query string (e.g. `?resourceName=frontdoor`)", http.StatusBadRequest)
```

Comment:
This should be a 412 than a 400. Should be able to use the `preconditionFailed` func here